lib/features/children/generosity_challenge_chat/chat_scripts/widgets/chat_input_field.dart

```diff
@@ -48,6 +48,21 @@ class _ChatInputFieldState extends State<ChatInputField> {
     switch (type) {
       case ChatScriptInputAnswerType.email:
         return Util.emailRegEx.hasMatch(input);
+      case ChatScriptInputAnswerType.phone:
+        final numericOnly = input.replaceAll(RegExp(r'[^\d]'), '');
```

Comment:
This code is copy/pasted. Should it be part of for example the Utils library so we only have it once?